taplt/ui/main_window.py

```diff
@@ -117,7 +117,15 @@ def __init__(self):
         self.toolBar = Toolbar(self)
         self.addToolBar(Qt.ToolBarArea.LeftToolBarArea, self.toolBar)
         self.toolBar.init_margins()
-        self.toolBar.init_actions(self)
+
+        # Toolbar setup actions for images, videos and whole slides
+        self.toolBar.init_actions('image', self.define_img_actions())
+        self.toolBar.init_actions('wsi', self.define_wsi_actions())
```

Comment:
I recommend changing 'wsi' into 'virtual_slide' or just 'slide' to make it more descriptive.